fixtures/xunit_tools.py

```diff
@@ -18,15 +18,36 @@
 }
 
 
+blacklist = [
+    'cfme/tests/containers/',
+    'cfme/tests/middleware/',
+    'cfme/tests/openstack/',
+    'utils/',
+    'rhos',
+    'rhev',
+    'hawkular',
+]
+compiled_blacklist = re.compile('(' + ')|('.join(blacklist) + ')')
+
+
 def pytest_addoption(parser):
-    # Create the cfme option group for use in other plugins
+    """Adds command line options."""
     parser.getgroup('cfme')
     parser.addoption("--generate-xmls", action="store_true", default=False,
         help="generate the xml files for import")
+    parser.addoption("--generate-legacy-xmls", action="store_true", default=False,
+        help="generate the legacy xml files for import")
+    parser.addoption("--xmls-testrun-id",
+        help="testrun id")
+    parser.addoption("--xmls-testrun-title",
+        help="testrun title")
+    parser.addoption("--xmls-apply-blacklist", action="store_true", default=False,
```

Comment:
We might want to switch this to `True` by default, otherwise one incorrect invocation will create test cases we don't care about that will have to be cleaned up.